lib/templates/erb/scaffold/_form.html.erb

```diff
@@ -0,0 +1,13 @@
+<%%= simple_form_for(@<%= singular_table_name %>) do |f| %>
```

Comment:
i believe the <%%= should be <%= for this on the simple form just using one %

Reply:
this file was actually generated from installing simple_form. i didn't write this. so perhaps this is how the file is created? with the "<%%="?